server/testing/fake-codex-app-server.ts

```
@@ -133,7 +133,14 @@ process.stdin.on("data", (chunk) => {
           break;
         }
         out({ jsonrpc: "2.0", id: msg.id, result: { ok: true } });
-        notify("item/started", { item: { id: "i1", type: "commandExecution", command: "ls -la" } });
+        const command = mode === "windows-command"
+          ? [
+              "\"C:\\WINDOWS\\System32\\WindowsPowerShell\\v1.0\\powershell.exe\"",
+              "-Command",
+              "\"Get-Content -Raw -LiteralPath 'C:\\Users\\Ada\\workspaces\\research\\NOTES.md'\"",
+            ].join(" ")
+          : "ls -la";
```

Biome (2.5.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)